model2vec/distill/distillation.py

```diff
@@ -125,7 +125,13 @@ def distill_from_model(
 
     model_name = getattr(model, "name_or_path", "")
 
-    config = {"tokenizer_name": model_name, "apply_pca": pca_dims, "apply_zipf": apply_zipf}
+    config = {
+        "tokenizer_name": model_name,
+        "apply_pca": pca_dims,
+        "apply_zipf": apply_zipf,
+        "hidden_dim": pca_dims,
```

Comment:
I think this is not correct. Because `pca_dims` can be "auto" or None. Instead, it should be `embeddings.shape[1]` or something?

Reply:
Good catch, fixed